src/components/Select/Select.tsx

```diff
@@ -26,25 +28,73 @@ interface Props {
    * Defines the style of the select.
    */
   style?: CSSProperties
+
+  /** Defines the custom error message of the input. */
+  isInvalidFeedback?: string
+  /** Defines the message for valid input. */
+  isValidFeedback?: string
+  /** Defines the default value */
+  defaultValue?: string | Array<string>
 }
 
 const Select = (props: Props) => {
-  const { value, multiple, isInvalid, disabled, size, onChange, children, className, style } = props
-
+  const {
+    defaultValue,
+    value,
+    multiple,
+    isValid,
+    isInvalid,
+    isInvalidFeedback,
+    isValidFeedback,
+    disabled,
+    size,
+    onChange,
+    children,
+    className,
+    style,
+  } = props
+  /**
+   * Defines the default style of the select.
+   */
+  const selectDefaultStyles = {
+    backgroundPosition: 'right calc(.375em + .55rem) center',
+    borderColor: isValid ? '#59b571' : isInvalid ? 'red' : 'black',
+  }
+  const feedbackDefaultStyle = {
+    color: isValid ? '#59b571' : isInvalid ? '#dc3545' : 'black',
+  }
```

Comment:
What is the reason for needing to define colors here? Can we rely on default bootstrap colors instead?

Reply:
I thought that it would be helpful to have a way to add some custom styles but wasn't sure where. If we are restricted to bootstrap I can use it but I think we should find a way to add our own custom css considering I think using only bootstrap limits styling improvements. 
But if i'm not mistakened we are still having issues testing styles with Jest so I'll remove these styles and leave the styling for another possible future issue.